src/Libraries/PythonNodeModelsWpf/PythonNode.cs

```diff
@@ -73,8 +73,14 @@ public void CustomizeView(PythonNode nodeModel, NodeView nodeView)
             nodeModel.DeletionStarted += NodeModel_DeletionStarted;
             nodeModel.Disposed += NodeModel_Disposed;
 
-            nodeView.PresentationGrid.Visibility = Visibility.Visible;
-            nodeView.PresentationGrid.Children.Add(new EngineLabel(nodeModel));
+            EngineLabel engineLabel = new EngineLabel(nodeModel);
+            Canvas.SetZIndex(engineLabel, 5);
+            engineLabel.HorizontalAlignment = HorizontalAlignment.Left;
+            engineLabel.VerticalAlignment = VerticalAlignment.Bottom;
+            engineLabel.Margin = new Thickness(14, -4, 0, 4);
+            nodeView.grid.Children.Add(engineLabel);
+            Grid.SetColumn(engineLabel, 0);
+            Grid.SetRow(engineLabel, 3);
```

Comment:
I guess these changes could be related to the regression I mentioned in comment

Reply:
Hi Aaron, 

yes, I moved this label to the node's 'grid' away from the 'PresentationGrid'.
Have just updated to fix this test.